binaryninjacore.h

```diff
@@ -163,127 +169,127 @@
 extern "C"
 {
 #endif
-	enum BNPluginLoadOrder
+	typedef enum BNPluginLoadOrder
 	{
 		EarlyPluginLoadOrder,
 		NormalPluginLoadOrder,
 		LatePluginLoadOrder
-	};
+	} BNPluginLoadOrder;
 
-	enum PluginLoadStatus
+	typedef enum PluginLoadStatus
```

Comment:
This one stood out to me for lacking the `BN` prefix. Is that something worth fixing?

Reply:
Quick check of the source shows this is ... not actually used in any APIs and only exists here to have parity between the UI plugin loader and the core plugin loader. I recommend Just Not Touching It And Things Will Be Fine.